catalog/core/catalog-core-ftp/src/main/java/ddf/catalog/core/ftp/FtpServerStarter.java

```diff
@@ -36,8 +40,16 @@
 
     private static final String DEFAULT_LISTENER = "default";
 
+    private static final String WANT_AUTH = "WANT";
```

Comment:
We currently trigger whether or not this is required for HTTP connections through the pax web config property: `org.ops4j.pax.web.ssl.clientauthwanted`. It might make sense to have a separate property in system.properties that the pax web config looked at as well so that we can toggle 2-way TLS for HTTP and FTP at the same time. I can't imagine that you'd have different rules for them if you're already in an environment that requires 2-way. Thoughts? @kcwire @pklinef